src/js/models/appUser.js

```diff
@@ -42,7 +42,7 @@ var AppUser = module.exports = BaseModel.extend({
             return Backbone.Model.prototype.save.apply(this, _.toArray(arguments)).then(function() {
                 this._lastPropertyValues = this.pick(AppUser.EDITABLE_PROPERTIES);
                 return arguments;
-            });
+            }.bind(this));
```

Comment:
The issue was with PhantomJS 1.x which runs our client-side tests. Might need to update to PhantomJS 2.x and all the related Grunt packges.